### DISCUSSION TITLE
How are the resources related to ESMValTool workshops stored / shared?

### DISCUSSION THREAD

**Discussion by ehogan** [ESMValGroup/Community#57] [General]
Hi @ESMValGroup/esmvaltool-developmentteam 👋 

How are the resources related to ESMValTool workshops  (e.g. slides) stored / shared? Were the resources for the workshop earlier this year stored somewhere shareable?

If there is no current solution to this, may I propose adding a `Workshops` (or equivalent) folder to the `Community` repository to store workshop-related resources?

What do you think? 😊

**Comment by LisaBock** [comment]
Hi @ehogan 
For some training events there is a link to the corresponding material in the table in the [Community/Readme](https://github.com/ESMValGroup/Community/blob/main/README.md). Is that what you looking for?

**Reply by ehogan** [reply]
Thank you very much for your response @LisaBock and apologies for the extremely late reply!

Yes, I was thinking we could do something like that, but for workshop content, and to host that workshop content in this repository, rather than on other (potentially external) web pages. Does that make sense? 😊

**Comment by remi-kazeroni** [comment]
> How are the resources related to ESMValTool workshops (e.g. slides) stored / shared? Were the resources for the workshop earlier this year stored somewhere shareable?

I believe slides and recordings were only made available to participants of workshops. I think so far only summaries of previous workshops were made publicly available through the website: https://www.esmvaltool.org/meetings.html

> If there is no current solution to this, may I propose adding a `Workshops` (or equivalent) folder to the `Community` repository to store workshop-related resources?

I think this is a good idea. The material is quite different from what we have for training activities so it's good to keep things separated.

**Reply by ehogan** [reply]
Thank you very much for your response @remi-kazeroni and apologies for the extemely late reply!

The link you shared no longer works; do you have an updated link, please?

Yes, I agree things should be kept separate. I would be happy to create a `Workshops` folder, if no one has any objections? 😊

**Reply by rswamina** [reply]
Would [here](https://github.com/ESMValGroup/Community) be a good idea?

**Reply by remi-kazeroni** [reply]
With the new website, some old links may have changed. You can find the list of previous workshops at: https://esmvaltool.org/meetings/ I think it would be goo to have a `Workshops` directory containing material. 

If needed, you could also propose changes to https://esmvaltool.org/meetings/ via https://github.com/ESMValGroup/ESMValTool-website

**Reply by ehogan** [reply]
Thanks everyone! I have opened an issue to do this: #94 😊
